pkgs/development/cuda-modules/setup-hooks/extension.nix

```diff
@@ -1,10 +1,5 @@
 final: _: {
-  # TODO:
```

Comment:
Don't we want to deprecate `cudaPackages.autoFixElfFiles` too?

Reply:
Forgot it was part of the refactor. I'll include it

Reply:
done